action.yml

```diff
@@ -16,6 +16,30 @@ inputs:
   environment-variables:
     description: 'Variables to add to the container. Each variable is of the form KEY=value, you can specify multiple variables with multi-line YAML strings.'
     required: false
+  environment-secrets:
+    description: 'Secrets to add to the container. Each secret is of the form KEY=value, you can specify multiple variables with multi-line YAML strings.'
+    required: false
+  family:
+    description: 'task-def family'
+    required: false
+  cpu:
+    description: 'CPU'
+    required: false
+  memory: 
+    description: 'Memory'
+    required: false
+  executionRoleArn:
+    description: 'executionRoleArn'
+    required: false
+  taskRoleArn:
+    description: 'taskRoleArn'
+    required: false
+  awslogs-group:
+    description: 'awslogs-group'
+    required: false
+  awslogs-region:
+    description: 'awslogs-region'
+    required: false
```

Comment:
Question; is the expectation here that some of this changes between github action execution? 

Until now, we haven't added these fields into the github action config because we supply a task def json and then we read the parameters that change and inline them into the task definition. But I'm wondering if there is a usecase that we've missed.

Reply:
Yeah some of these might not be needed like the logs, and task arns since you can just use the names. To be honest I think the biggest item needed is secret arns since they do contain the AWS Account Id and would prefer not to commit that in the repo.